src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is {person.name}
+    </h2>
+    <p className="Person__age">
+      {person.age ? `I am ${person.age}` : `` }
+    </p>
```

Comment:
1. you shoudn't show empty paragraph if person hasn't age property
2. Check, pls, your test, why did toy ignore this point?
3. Where is your DEMO link?
4. fix, pls, my prev comment